CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.36.2
+* Bug - fix windows logfile writing [#2347](https://github.com/aws/amazon-ecs-agent/pull/2347)
+* Bug - update sbin mount point to avoid conflicct with Docker [#2345](https://github.com/aws/amazon-ecs-agent/pull/2337)
```

Comment:
s/conflicct/conflict/

Also can we call out which version of docker this addresses? If that’s available readily. 

“avoid conflict with docker 19.blah”